### PR TITLE
Update quickstart docs with brew install cmd

### PR DESCRIPTION
**Pull request by adam2k**
Fixes https://github.com/astronomer/issues/issues/1324

### CODE REVIEW

File: v0.14/cli-quickstart.md [hunk 1]
```diff
@@ -15,50 +15,59 @@ To install the CLI, make sure you've installed:
...
-  help            Help about any command
-  upgrade         Check for newer version of Astronomer CLI
-  user            Manage astronomer user
-  version         Astronomer CLI version
-  workspace       Manage Astronomer workspaces
-
-Flags:
-  -h, --help   help for astro
-
-Use "astro [command] --help" for more information about a command.
```
Comment (andriisoldatenko) [inline v0.14/cli-quickstart.md:61]: @adam2k why did you remove this?

File: v0.14/cli-quickstart.md [hunk 2]
```diff
@@ -15,50 +15,59 @@ To install the CLI, make sure you've installed:
...
+curl -sSL https://install.astronomer.io | sudo bash -s -- v0.7.5
+```
+
+> Note: if you're running macOS Catalina and beyond and are trying to install our CLI, do the following:
+
+Run `sudo -K` to reset/get back to un-authenticated then run the following to install the CLI properly:
+
+```
+curl -sSL https://install.astronomer.io | sudo bash -s < /dev/null
+```
```
Comment (paolaperaza) [inline v0.14/cli-quickstart.md:54]: Would maybe change format/wording to:

> Note: If you're running macOS Catalina and beyond,
>
> 1. Run `sudo -K` to reset/get back to un-authenticated \
> 2. Run the following to install the CLI properly:
>
> 
> ```
> curl -sSL https://install.astronomer.io | sudo bash -s < /dev/null
> ```

File: v0.14/cli-quickstart.md [hunk 3]
```diff
@@ -15,50 +15,59 @@ To install the CLI, make sure you've installed:
...
+
 **Note:** The `curl` command will work for Unix (Linux+Mac) based systems. If you want to run on Windows 10, you'll need to run through [this guide](https://www.astronomer.io/docs/cli-installation-windows-10) on getting Docker for WSL working.
 
 ### Confirm the Install
 
 Let's make sure that you have Astro CLI installed on your machine and have a project to work from.
 
 ```bash
-$ astro
+$ astro version
```
[the remark this answers is not in the document]
Comment (andriisoldatenko) [reply v0.14/cli-quickstart.md:63]: absolutely nope, it was correct :)
when you run `astro` without arguments, you can see help message. (default behavior for all commands)

[the remark this answers is not in the document]
Comment (andriisoldatenko) [reply v0.14/cli-quickstart.md:63]:
```
astro
astro is a command line interface for working with the Astronomer Platform.

Usage:
  astro [command]

Available Commands:
  auth            Manage astronomer identity
  cluster         Manage Astronomer EE clusters
  completion      Generate autocompletions script for the specified shell (bash or zsh)
  config          Manage astro project configurations
  deploy          Deploy an airflow project
  deployment      Manage airflow deployments
  dev             Manage airflow projects
  help            Help about any command
  upgrade         Check for newer version of Astronomer CLI
  user            Manage astronomer user
  version         Astronomer CLI version
  workspace       Manage Astronomer workspaces

Flags:
  -h, --help   help for astro

Use "astro [command] --help" for more information about a command.

```

File: v0.14/cli-quickstart.md [hunk 4]
```diff
@@ -15,50 +15,59 @@ To install the CLI, make sure you've installed:
 
 - [Docker](https://www.docker.com/) (v18.09 or higher)
 
-## Install
+## Installation
+
+### Mac OSX
```
[the remark this answers is not in the document]
Comment (adam2k) [reply v0.14/cli-quickstart.md:20]: ah ok, I can update this.

File: v0.14/cli-quickstart.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (adam2k) [reply v0.14/cli-quickstart.md:63]: @andriisoldatenko I was removing that because it's a lot of unnecessary.  text to add into the documentation.  If we tell them to run `astro version` then we only have to add that they should see: 
```
Astro CLI Version: 0.15.0
Git Commit: c4fdeda96501ac9b1f3526c97a1c5c9b3f890d71
```
Instead of the entire help message that prints when you run `astro`.

File: v0.14/cli-quickstart.md (hunk 1 above)
Comment (adam2k) [reply v0.14/cli-quickstart.md:61]: It was a lot of extra text to add into the doc.  If we give them a different command to run as a test then we don't need to include all of the help text output.  We could make a mention about running `astro help` to see all the commands or something too?

File: v0.14/cli-quickstart.md [hunk 5]
```diff
@@ -15,50 +15,59 @@ To install the CLI, make sure you've installed:
 
 - [Docker](https://www.docker.com/) (v18.09 or higher)
 
-## Install
+## Installation
+
```
[the remark this answers is not in the document]
Comment (adam2k) [reply v0.14/cli-quickstart.md:19]: I'll make those changes.  Windows doesn't work with Homebrew, so I'll move that comment up.

File: v0.14/cli-quickstart.md (hunk 1 above)
Comment (andriisoldatenko) [reply v0.14/cli-quickstart.md:61]: the idea if this "extra" text is to show for user after installation, where to go next and test `astro` installation in same time.

Comment (andriisoldatenko) [reply v0.14/cli-quickstart.md:61]: similar too all tools in internet: 
- https://github.com/heroku/cli#command-topics or so.
